test cases/common/105 find program path/program.py

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
These shebangs don't need to be changed. On windows, the `py` script runner will correctly invoker Python 3 if this is set to `#!/usr/bin/env python3`.